stock_picking_completion_info/models/stock_picking.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, models, fields
+
+
+class PickingType(models.Model):
+
+    _inherit = 'stock.picking.type'
+
+    display_completion_info = fields.Boolean(
+        help='Inform operator of a completed operation at processing and at'
+        ' completion'
+    )
+
+
+class StockPicking(models.Model):
+
+    _inherit = 'stock.picking'
+
+    completion_info = fields.Selection(
+        [
+            ('no', 'No'),
+            (
+                'last_picking',
+                'Completion of this operation allows next operations to be '
+                'processed.',
+            ),
+            (
+                'next_picking_ready',
+                'Next operations are ready to be processed.',
+            ),
+        ],
```

Comment:
To add:
"Full order picking: You are processing a full order picking that will allow next operation to be processed."
In blue would be nice